app.rb

```diff
@@ -1,8 +1,32 @@
 require 'sinatra/base'
+require_relative './lib/peep'
 
 class Chitter < Sinatra::Base
-  get '/test' do
-    'Test page'
+
```

Comment:
This is all pretty neat and it's great that you're thought to encapsulate some of the database logic in the `select_database` and `connect` methods.

The main things to reflect on in terms of improvements would be RESTful routing and the user journey.

`GET '/peeps'` and `POST '/peeps'` are both RESTful but `GET '/post'` is not, since you don't have a resource called `post`. The conventional route would be `GET '/peeps/new'` (for the route which returns the form to create a new `Peep`).

Regarding the user journey – at the moment, the user is redirected to the new peep form, from the `/` path, which may feel a bit odd. I'd probably expect to either see a welcome screen or the list of all peeps.  Interested to hear your thoughts on that though!